telegram/ext/messagequeue.py

```diff
@@ -283,40 +268,36 @@ def __call__(self, promise, is_group_msg=False):
 
 
 def queuedmessage(method):
-    '''A decorator to be used with `telegram.bot.Bot` send* methods.
+    """
+    A decorator to be used with :attr:`telegram.Bot` send* methods.
 
     Note:
-        As it probably wouldn't be a good idea to make this decorator a
-        property, it had been coded as decorator function, so it implies that
-        **first positional argument to wrapped MUST be self**\.
+        As it probably wouldn't be a good idea to make this decorator a property, it has been coded
+        as decorator function, so it implies that first positional argument to wrapped MUST be
+        self.
 
     The next object attributes are used by decorator:
 
     Attributes:
-        self._is_messages_queued_default (:obj:`bool`): Value to provide
-            class-defaults to `queued` kwarg if not provided during wrapped
-            method call.
-        self._msg_queue (:obj:`telegram.ext.messagequeue.MessageQueue`):
-            The actual `MessageQueue` used to delay outbond messages according
-            to specified time-limits.
+        self._is_messages_queued_default (:obj:`bool`): Value to provide class-defaults to
+            ``queued`` kwarg if not provided during wrapped method call.
+        self._msg_queue (:class:`telegram.ext.messagequeue.MessageQueue`): The actual
+            ``MessageQueue`` used to delay outbound messages according to specified time-limits.
 
     Wrapped method starts accepting the next kwargs:
 
     Args:
-        queued (:obj:`bool`, optional): if set to ``True``, the `MessageQueue`
-            is used to process output messages.
-            Defaults to `self._is_queued_out`.
-        isgroup (:obj:`bool`, optional): if set to ``True``, the message is
-            meant to be group-type (as there's no obvious way to determine its
-            type in other way at the moment). Group-type messages could have
-            additional processing delay according to limits set in
-            `self._out_queue`.
-            Defaults to ``False``.
+        queued (:obj:`bool`, optional): if set to ``True``, the ``MessageQueue`` is used to process
+            output messages. Defaults to `self._is_queued_out`.
+        isgroup (:obj:`bool`, optional): if set to ``True``, the message is meant to be group-type
+            (as there's no obvious way to determine its type in other way at the moment).
+            Group-type messages could have additional processing delay according to limits set
+            in `self._out_queue`. Defaults to ``False``.
 
     Returns:
-        Either :obj:`telegram.utils.promise.Promise` in case call is queued,
-        or original method's return value if it's not.
-    '''
+        ``telegram.utils.promise.Promise``: in case call is queued or original method's return
```

Comment:
Promise is undocumented atm, and IMO it is good that way since it's an internal mechanism.